src/Analysis/Engine/Test/CompletionTests.cs

```diff
@@ -443,19 +443,19 @@ class B(A):
 
         [TestMethod, Priority(0)]
         public async Task TopLevelCompletions() {
-            using (var s = await CreateServerAsync(PythonVersions.LatestAvailable3X)) {
-                var uri = GetDocument(Path.Combine("TestData", "AstAnalysis", "TopLevelCompletions.py"));
-                await s.LoadFileAsync(uri);
+            using (var server = await CreateServerAsync(PythonVersions.LatestAvailable3X)) {
+                var uri = GetDocument(@"TestData\AstAnalysis\TopLevelCompletions.py");
```

Comment:
I don't think this path should be getting changed back to `\`.

Reply:
Sorry, merge conflict.